docs/src/literate-howto/postprocessing.jl

```diff
@@ -129,6 +129,12 @@ Plots.plot(getindex.(points,1), u_points, xlabel="x (coordinate)", ylabel="u (te
 Plots.plot(getindex.(points,1), getindex.(q_points,1), xlabel="x (coordinate)", ylabel="q_x (flux in x-direction)", label=nothing)
 # *Figure 4*: ``x``-component of the flux along the cut line from *Figure 2*.
 
+#md # ## What next?
+#md # For more complicated visualization workflows we recommend either using our visualization tool [FerriteViz.jl](https://github.com/Ferrite-FEM/FerriteViz.jl)
+#md # or users should export the solution into vtk files and use e.g. [ParaView](https://www.paraview.org/), [Mayavi](https://docs.enthought.com/mayavi/mayavi/), ... .
+#md # It should be noted that the ordering of the DofHandler and the numbering of the nodes does not match, hence we cannot directly use solution
+#md # vectors to assign colors to discretizations.
```

Comment:
The part on VTK is already mentioned [above](https://ferrite-fem.github.io/Ferrite.jl/previews/PR875/howto/postprocessing/#Exporting-to-VTK). Would it be better to rename this subsection and write something like
 ```suggestion
#md # ## Visualizing grid data in Julia
#md # To visualize the solution on the grid, note that the numbering of the degrees of
#md # freedom generated by the `DofHandler` does not match the grid node ordering. 
#md # To get data at the nodes, the function [`evaluate_at_grid_nodes`](@ref) can be used. 
#md # For more complicated visualization workflows in Julia we recommend using our visualization
#md # tool [FerriteViz.jl](https://github.com/Ferrite-FEM/FerriteViz.jl)
```
(Haven't checked that if the links work)

Reply:
Not sure. Above we only refer to the export of the fluxes, where we don't have a good way to compute them correctly in ParaView.

Reply:
Yes, but the regular export was already shown in the heat equation example that this how-to includes. In that case, perhaps it would make sense to modify that case above to say that one can then export the fluxes in addition to the temperature field, and modify that code block to 
```julia
vtk_grid("heat_equation", grid) do vtk
    vtk_point_data(vtk, dh, u)
    vtk_point_data(vtk, projector, q_projected, "q")
end;
```
if that resolves that. But I think for new users the vtk export option is quite visible since it is used in most (all?) examples.

Reply:
Yea, but still users come to the conclusion that they better write their own visualizer over and over again, hence I wanted to really highlight that it is easier to use what is there already. :)